lab-devon/lib/read-files.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const fs = require('fs');
+
+module.exports = function(filePath, callback){
+  let result = [];
+  fs.readFile(filePath[0], (err, data) => {
+    if (err) return callback(err);
+    result.push(data.toString());
+
+    fs.readFile(filePath[1], (err,data) => {
+      if (err) return callback(err);
+      result.push(data.toString());
+
+      fs.readFile(filePath[2], (err,data) => {
+        if (err) return callback(err);
+        result.push(data.toString());
+      });
+    });
+  });
+  callback(null, result);
```

Comment:
When I run ```node index.js <args>``` in my terminal, I get back an empty array. To solve this, you need to move ``` callback(null, result);``` right after line 17 because it stays inside within your third ```fs.readFile``` function that happens on line 15. 

This ensures that you have pushed all three text outputs into your ```result``` array before it gets returned in the callback.